iot-hub/Samples/service/RoleBasedAuthenticationSample/RoleBasedAuthenticationSample.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.Azure.Devices;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RoleBasedAuthenticationSample
+{
+    /// <summary>
+    /// This sample connects to the IoT hub using Azure active directory credentials and sends a cloud-to-device message.
+    /// </summary>
+    public class RoleBasedAuthenticationSample
+    {
+        public async Task RunSampleAsync(ServiceClient client, string deviceId)
+        {
+            Console.WriteLine("Connecting using token credential.");
+            await client.OpenAsync();
+            Console.WriteLine("Success.");
```

Comment:
nit: good to restate logging as Success opening connection or something of that sort